test/planerText.test.coffee

```diff
@@ -367,3 +375,9 @@ describe 'planer#extractFromPlain', ->
   it 'does not throw errors on messages with malformed links', ->
     msgBody = "http://test.lever.co/YOU HAVE AN INTERVIEW TODAY\nhttps://test.lever.co/interviews/07a605a0-0d0a-00e8-00aa-f02ca5350180 is coming up today athttps://www.google.com/calendar/event?eid=Z2FrbzhxcW0000YwbmtmMDN1ZWZ2OHAycnMgbGV2Z0000W1vLmNvbV82am00000000hvY3RjN200000000Vjc00000Bn.\n\nhttps://test.lever.co/interviews/0000a5ab-000b-43aa-a00a-f020003aaa84"
     expect(planer.extractFromPlain(msgBody)).to.equal(msgBody)
+
+  it 'does not take longer than expected when processing inline replies that do not end with a marker', ->
```

Comment:
why not assert processing time? like ensure it's less than 10ms or something.

Reply:
Can it possibly result in a flaky test?

Reply:
Hrmm yes I am relying on the default timeout of 2 seconds here. Do you think it is sufficient to set a specific timeout, or should I compute the timing and assert that specifically?

Reply:
I would compute the timing, that way we get a nice test failure like "should have completed in 50ms" vs a test timeout which can appear as a normal flaky test and not an issue.